webui/react/src/pages/ExperimentDetails/CodeViewer/CodeViewer.tsx

```diff
@@ -342,33 +342,46 @@ const CodeViewer: React.FC<Props> = ({
                   */
                   <Tooltip title="Download File">
                     {
-                      !String(activeFile.key).includes('Configuration') && (
-                        // hiding the download for configs until next iteration
-                        <DownloadOutlined
-                          className={css.noBorderButton}
-                          onClick={(e) => {
-                            const filePath = String(activeFile.key);
-                            if (filePath.includes('Configuration')) {
-                              const url = filePath.includes('runtime')
-                                ? URL.createObjectURL(new Blob([ runtimeConfig ]))
-                                : URL.createObjectURL(new Blob([ submittedConfig as string ]));
-
-                              handlePath(e, {
-                                external: true,
-                                path: url,
-                              });
-                            } else {
-                              handlePath(e, {
-                                external: true,
-                                path: paths.experimentFileFromTree(
-                                  experimentId,
-                                  String(activeFile.key),
-                                ),
-                              });
-                            }
-                          }}
-                        />
-                      )
+                      <DownloadOutlined
```

Comment:
I don't know if you had conversation about design, but IMO I think we should use menu here instead of just icon for future use, such as copy file content etc... what do you think?